app/views/documents/show/_content_settings.html.erb

```diff
@@ -42,7 +42,12 @@
 <% items << {
   field: t("documents.show.content_settings.gets_history_mode.title"),
   value: @edition.political? ? t("documents.show.content_settings.gets_history_mode.true_label") : t("documents.show.content_settings.gets_history_mode.false_label"),
```

Comment:
Should this also be made into an `if` block for consistency as per the code change below?

Reply:
That's a good shout, it's a very long line.